packages/@react-aria/dialog/docs/useDialog.mdx

```diff
@@ -72,6 +72,9 @@ to create modal dialogs, popovers, and other types of overlays.
 If a dialog does not have a visible title element, an `aria-label` or `aria-labelledby`
 prop must be passed instead to identify the element to assistive technology.
 
+`useDialog` applies 'aria-modal' to the modal. This doesn't cover every browser/screen reader combination and should be combined with `@react-aria/aria-modal-polyfill`, see the [Aria Modal Polyfill docs](../react-aria/aria-modal-polyfill.html).
```

Comment:
I'd move this after the last paragraph before the example below which talks about this already.

```suggestion
**Note:** `useModal` only hides content within parent `OverlayProvider` components. However, if you have additional content in your application outside any `OverlayProvider`, then you should use the `@react-aria/aria-modal-polyfill` package to ensure that this content is hidden while modals are open as well. See the [aria-modal-polyfill docs](../react-aria/aria-modal-polyfill.html) for more information.
```